app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,14 @@
     <title>WNB.rb</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Rubik:200,300,400,500,600,700,800,900">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=<%= ENV['GA_ID'] %>"></script>
```

Comment:
The lines of code added here come directly from the Google Analytics installation instructions.